server/testdriver/junit4/src/test/java/InfinispanRuleBasicTest.java

```diff
@@ -0,0 +1,30 @@
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.configuration.ClientIntelligence;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerRuleBuilder;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class InfinispanRuleBasicTest {
+   @ClassRule
+   public static final InfinispanServerRule SERVER = InfinispanServerRuleBuilder.server();
+
+   @Rule
+   public InfinispanServerTestMethodRule SERVER_TEST = new InfinispanServerTestMethodRule(SERVER);
+
+   @Test
+   public void testSingleServer() {
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      builder.clientIntelligence(ClientIntelligence.BASIC);
+      RemoteCache<String, String> cache = SERVER_TEST.hotrod().withClientConfiguration(builder).withCacheMode(CacheMode.DIST_SYNC).create();
+            cache.put("k1", "v1");
```

Comment:
Wrong indent, and the previous line should be wrapped

Reply:
oki